### PR TITLE
Add npm prepare script

### PR DESCRIPTION
**Pull request by t-richard**
This is PR adds a `prepare` script to the package.json.

This is a special npm script that is executed when installing a package from a git URL like so

```bash
npm install git@github.com:t-richard/lift.git#add-prepare-script
```

This would come in handy for the community to test WIP/draft constructs and is much more user-friendly than using `npm link`.

Previously, doing this would not work because we would be missing the `dist` folder in ` node_modules/serverless-lift`

The script behaviour is described at https://docs.npmjs.com/cli/v7/using-npm/scripts#life-cycle-scripts

It also says
> Runs BEFORE the package is published

This is why I also deleted `npm run build` from the release GH Action workflow

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -78,7 +78,8 @@
     "type": "tsc --noEmit",
     "build": "etsc",
     "watch": "nodemon",
-    "test": "jest"
+    "test": "jest",
+    "prepare": "etsc"
```
Comment (fredericbarthelet) [inline package.json:82]:
```suggestion
    "prepare": "build"
```

This way, if we'd change in the future the build step for the project, `prepare` would be updated as well :)

**Review by fredericbarthelet — CHANGES_REQUESTED** [review]
Looks  like a nice improvement :) ! Thanks @t-richard !
I suggested a minor improvement to ensure coherence between build and prepare. Let me know what you think of it.

**Comment by t-richard** [comment]
@fredericbarthelet that makes sense. I wasn't able to make it work with your suggestion. 

Just using "build" is not enough apparently, all solutions I found online suggested to use the full command like `npm run build` so this is what I did and it works as expected.

**Review by fredericbarthelet — APPROVED** [review]
LGTM. Thanks for your contribution @t-richard :) !

**Pull request merged**